products/editions/enterprise-edition/b2b-suite/concepts/system-architecture.md

```diff
@@ -0,0 +1,60 @@
+# System architecture
+
+The B2B-Suite is a collection of loosely coupled mostly uniform **components** packaged with a small example plugin and a common library.
+
+## Component layering
+
+A single component with all layers and the maximum of allowed dependencies looks like this:
+
+![image](../../../../.gitbook/assets/b2b-architecture-component.svg)
+
+The responsibilities from bottom to top:
+
+| Layer       | Description                                                                                                                                                                                                      |
+|-------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Shop-Bridge | Bridges the broad Shopware interfaces to the specific framework requirements <ul><li>Implements interfaces provided by the framework</li><li>Subscribes to shopware events and calls framework services</li><ul> |
```

Comment:
![Bildschirmfoto 2022-04-26 um 11 28 38](https://user-images.githubusercontent.com/23453578/165269069-b7a55723-af0a-41bb-b13f-6a80d4331325.png)

The list is broken.